src/Form/MiradorConfigForm.php

```diff
@@ -65,4 +98,28 @@ protected function getEditableConfigNames() {
     ];
   }
 
+
```

Comment:
I suspect PHPCS might flag this, due to the two empty consecutive lines? Not sure.

Reply:
Good catch, removed.